embedded/src/main/java/org/wildfly/core/embedded/EmbeddedStandaloneServerFactory.java

```diff
@@ -341,10 +341,7 @@ public void stop() {
 
         @Override
         public String getProcessState() {
-            if (currentProcessState == null) {
-                return null;
-            }
-            return currentProcessState.toString();
+            throw new UnsupportedOperationException("The access to the internal process state is unsupported for this standalone embedded server");
```

Comment:
If you put a message here it really should be internationalized.

A plain no-message UOE with a code comment should be enough. I think it would be a developer messing around who would hit this, not a user. They can analyze the stack trace find the code and read the comment.

Reply:
Understood, thanks!, yes that was the intention, I throw the UOE for a developer point of view only. Fixed.